integration_test/warehouse/integration_test.go

```diff
@@ -1233,6 +1233,8 @@ func TestUploads(t *testing.T) {
 			<-done
 		})
 		t.Run("allowMerge=false", func(t *testing.T) {
+			t.Skipf("TODO: reintroduce with append vs merge")
```

Comment:
This diff looks weird :/

Reply:
Most probably a UI bug

Reply:
I could not find the commit [9c860c6](https://github.com/rudderlabs/rudder-server/pull/4140/commits/9c860c6f7964fb111264061a8ebd7ac7630820a5) on master. Did you merge with a merge commit the last time you sync?

Reply:
I think this is causing the diff to happen https://github.com/rudderlabs/rudder-server/commit/8fb633d369e241851555462596a8054866233158, since it is a different hash

Reply:
yes we shouldn't worry about it

Reply:
we might need to use a second merge commit after all for the integration tests :/